File: examples/Person.cs
class Person{
    // Propiedades Privadas
    private string name
    private int age
    // Metodos Publicos
    // constructor
    public Person(string name, int age){
        this.name = name;
        this.age = age;
    }
    // Getters
    public string getName(){
        return name;
    }
    public int getAge(){
        return age;
    }
    // Setters
    public void setName(name){
        this.name = name;
    }
    public void setAge(age){
        this.age = age;
    }
}
// Instancia
const user = new Person("cristian",18)
user.setAge(33)

File: examples/Person.cpp
class Person {
private: // Propiedades Privadas
    string name;
    int age;
public: // Metodos Publicos
    // Constructor
    Person(string name, int age){
        this->name = nombre;
        this->age = apellido;
    }
    // Getters
    string getName() {
        return name;
    }
    int getAge() {
        return age;
    }
    // Setters
    void setName(string name) {
        this->name = name;
    }
    void setAge(int age) {
        this->age = age;
    }
};
// Instancia
Person user("cristian", 18);
user.setAge(33);